README.md

```diff
@@ -19,7 +19,15 @@ To build and serve the site, you'll need these tools:
 
 ### Setup
 
-Once you've installed the [prerequisites](#prerequisites), get local packages:
+The etcd.io website uses the [Docsy](https://www.docsy.dev) Hugo theme. Install the `git` submodule with the following command:
+
+# pull in the Docsy submodule
```

Comment:
Sorry, on second thought I'm not sure we need this copy at all (we're already saying to run a command in the previous line)